Description: Fix FTBFS on GNU/Hurd
 On Hurd, due to a missing implementation _POSIX_TIMERS = 0,
 _POSIX_MONOTONIC_CLOCK [0] and _POSIX_CLOCK_SELECTION = 200809L. Such
 combination is considered non POSIX-compliant by vlc upstream which
 don't accept patches to workaround it. More info at [1,2].
 .
 [0] http://anonscm.debian.org/viewvc/pkg-glibc/glibc-package/trunk/debian/patches/hurd-i386/local-clock_gettime_MONOTONIC.diff?revision=5951&view=markup
 [1] https://lists.debian.org/debian-hurd/2014/02/msg00112.html
 [2] https://lists.debian.org/debian-hurd/2014/02/msg00118.html
Author: Gabriele Giacone <1o5g4r8o@gmail.com>
Forwarded: not-needed
--- a/src/posix/thread.c
+++ b/src/posix/thread.c
@@ -300,7 +300,7 @@ void vlc_cond_init (vlc_cond_t *p_condva
 
     if (unlikely(pthread_condattr_init (&attr)))
         abort ();
-#if (_POSIX_CLOCK_SELECTION > 0)
+#if (_POSIX_TIMERS > 0) && (_POSIX_CLOCK_SELECTION > 0)
     vlc_clock_setup ();
     pthread_condattr_setclock (&attr, vlc_clock_id);
 #endif
@@ -874,7 +874,7 @@ mtime_t mdate (void)
  */
 void mwait (mtime_t deadline)
 {
-#if (_POSIX_CLOCK_SELECTION > 0)
+#if (_POSIX_TIMERS > 0) && (_POSIX_CLOCK_SELECTION > 0)
     vlc_clock_setup ();
     /* If the deadline is already elapsed, or within the clock precision,
      * do not even bother the system timer. */
@@ -901,7 +901,7 @@ void msleep (mtime_t delay)
 {
     struct timespec ts = mtime_to_ts (delay);
 
-#if (_POSIX_CLOCK_SELECTION > 0)
+#if (_POSIX_TIMERS > 0) && (_POSIX_CLOCK_SELECTION > 0)
     vlc_clock_setup ();
     while (clock_nanosleep (vlc_clock_id, 0, &ts, &ts) == EINTR);
 
